### DISCUSSION TITLE
Hires file scrubbing preview speed

### DISCUSSION THREAD

**Discussion by r3538987** [mifi/lossless-cut#1594] [Q&A]
Had someone noticed that low resolution (SD) files display preview quickly when moving position over timeline, but once files are HD, preview is noticeable slower? 
In example in AviDemux image preview is almost instant, which makes searching for particular position much easier.

**Comment by AmoreMio8** [comment]
Yes. I would also like the fast forward option in "avidemux". By holding down the right arrow key on my keyboard, I can see quick previews at 6 second intervals. 
To do this with losslesscut I have to constantly turn the mouse wheel. However, as we approach the end of the video, its responsiveness decreases. 
 It would be great if a fast forward or preview feature comes in like in Avidemux.

**Comment by r3538987** [comment]
After investigating more, mine previous issue can be reproduced only with WMV/ASF files. 
When importing such files, app even mentioned that it's not natively supported, meaning such issues can occur.
Once you use software with MP4 or MKV files, scrubbing through timeline shows previews quickly* (depends on codec).

**Reply by mifi** [reply]
You're right. for unsupported formats like wmv/asf, a ffmpeg-backend preview will be created, which is quite slow unfortunately. More info in FAQ and #88
